src/atoms/breadcrumbs/Breadcrumbs.stories.js

```diff
@@ -40,7 +51,10 @@ storiesOf('Atoms/Breadcrumbs', module)
           </span>
         </template>
         <template #current="data">
-          <span style="padding: 0 10px; color: #2962ff; line-height: 1.5;">
+          <span
+            aria-current="page"
+            style="padding: 0 10px; color: #2962ff; line-height: 1.5;"
```

Comment:
Imo we should remove line-height in all places in this component, it could be problematic in the future :)